src/transaction/log_tran_table.c

```diff
@@ -3562,6 +3559,36 @@ logtb_tran_update_unique_stats (THREAD_ENTRY * thread_p, BTID * btid, int n_keys
   return error;
 }
 
+// *INDENT-OFF*
+int
+logtb_tran_update_unique_stats (THREAD_ENTRY * thread_p, const BTID &btid, const btree_unique_stats &ustats,
+                                bool write_to_log)
+{
```

Comment:
check unique constraint violation

Reply:
I considered checking here, but not all current users of this function check for unique constraint violation; I want to postpone adding the check in the next refactoring patch that will target transaction unique statistics specifically.